docs/podman-stats.1.md

```diff
@@ -4,7 +4,7 @@
 podman\-stats - Display a live stream of 1 or more containers' resource usage statistics
 
 ## SYNOPSIS
-**podman** **stats** [*options* [...]] [container]
+**podman** **stats** [*options*] [*container*]
```

Comment:
Why is this one different? Should be
`**podman stats** [*options*] [*container*]